docs/data/base/getting-started/customization/customization.md

```diff
@@ -52,6 +52,40 @@ In this case, `MyCustomThumb` component receives the `ownerState` object with th
 
 You can use this object to style your component.
 
+## Customizing slots' props
+
+The `slotProps` prop may be used to customize the props of the inner components.
+The most common use case is setting a class name, but you can set any prop, including event handlers.
+
+The following example shows how to add a custom class to two of the Unstyled Switch's slots:
+
+```tsx
+function Switch(props: SwitchUnstyledProps) {
+  const slotProps = {
+    thumb: {
+      className: 'switch:thumb',
```

Comment:
```suggestion
      className: 'thumb',
```

How about removing `switch:` from the classnames from this snippet? It's more consistent with the demo below (where the classes are just `.thumb` and `input`), and the `:` made me wonder for a second if its some special syntax 🤔

Reply:
I wanted to highlight that you're free to choose whatever class you'd like here. Perhaps the use of a colon may be confusing. Do you think classes like `my-thumb` would work better?

Reply:
Yeah I think `my-thumb` looks more intuitive!